darts/tests/test_metrics.py

```diff
@@ -30,15 +32,22 @@ def test_zero(self):
         with self.assertRaises(ValueError):
             metrics.mape(self.series1, self.series1)
 
+        with self.assertRaises(ValueError):
+            metrics.smape(self.series1, self.series1)
+
```

Comment:
Similarly to my comment on @pennfranc PR I believe that we should split these tests and try to keep a single test for a single assert or at least a single test for a single failure reason. (each test method name should describe what it is testing). The problem with the current tests is that if mape is failing we have no idea if it is only mape or other methods will also fail like smape also I believe the reason for failing is not very clear each time. Let me know what you think :)